### PR TITLE
[WIP]Reimplement Stripe form

### PR DESCRIPTION
**Pull request by marizaswart**
### Description
The stripe form had to be reimplemented due to removal from the code with another PR.
To clarify the difference between the stripe payment and the order(checkout) process, the decision was made to change the naming to 'charge' instead of 'cart'.

### What have learnt
* To define proper naming during the planning phase to avoid clashes during the development process

### CODE REVIEW

**Comment by ScottyyyB** [comment]
LGTM

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -31,14 +31,14 @@ def page_path_from(page_name)
     when 'thaitanic' then restaurant_path(Restaurant.find_by(name: page_name))
     when 'landing' then root_path
     when 'sign up' then new_user_registration_path
-    when 'cart' then charges_path()
+    when 'checkout' then charges_path()
```
Comment (diraulo) [inline features/step_definitions/basic_steps.rb]: the parenthesis aren't really necessary here

File: features/step_definitions/basic_steps.rb [hunk 2]
```diff
@@ -31,14 +31,14 @@ def page_path_from(page_name)
...
   end
 end
 
 Then("I should be on the {string} page") do |page|
   sleep(5)
   case page
```
Comment (diraulo) [inline features/step_definitions/basic_steps.rb]: Why are we using a case statement if we only have a single case? I'd re-use the `page_path_from` here to get the path to test for.

**Review by diraulo — CHANGES_REQUESTED** [review]
Few minor things to look at before we merge this

File: features/step_definitions/basic_steps.rb (hunk 2 above)
Comment (marizaswart) [reply features/step_definitions/basic_steps.rb]: Will fix this :)

**Pull request closed without merging**